### PR TITLE
Fix #2859: Replace 'https upgrades' with 'data saved' stat.

### LINKED ISSUE #2859: HTTPS vs Data Saved
### Description: 
Like Android and Desktop, please replace HTTPS with Data Saved on the NTP
 
Text to be

**Est Data Saved**

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #2859 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Test Plan:

- Visit few websites with ads
- Verify that data saved stat is being updated

## Screenshots:

Zrzut ekranu 2021-01-22 o 16 29 17

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "stats": {
             "ads": "0xFB542B",
             "trackers": "0x742BC4",
-            "httpse": "0xA0A5EB",
+            "dataSaved": "0xA0A5EB",
             "timeSaved": "0xffffff"
```
Comment (kylehickinson) [inline Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json:31]: Are we sure we want to separate these? Will potential iOS 14 widgets or detailed stats screens want to show a HTTPSE stat?

Comment (iccub) [reply Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json:31]: I do not think we ever go back to httpse stat, i'll ask,
Do you fetch from this json in your widget code btw?

**Review by Brandon-T — APPROVED** [review]

File: Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json (hunk 1 above)
Comment (kylehickinson) [reply Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json:31]: No, not from JSON right now: 

https://github.com/brave/brave-ios/commit/014029b529ba199890799792c43f68fc57564f1f#diff-0f101c2a123f5e274262466a7c30ad4789da612782cf696ec5e1fa0c05066f8cR24-R35

But cool, if we're not planning on bringing back HTTPSE stats ever again we can maybe also remove HTTPSE tracking code as well (and HTTPSE lib)

Comment (iccub) [reply Client/Assets/Themes/ACE618A3-D6FC-45A4-94F2-1793C40AE927.json:31]: Talked to James, no plans to show the stat.
But the httpse detection code is still used to show one educational popup, i think we have to keep it for now

**Pull request merged**